### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by CakeCarrot** [pau101/Wings#89]
4 winged angel wings, dragonfly wings, Beetle wings

**Comment by LordGriefington** [comment]
good Idea - I would have many ideas for wings, demon wings (2,3 winged versions)
- Crystal wings in different colours (inspired by Touhou), bird wings
- if you something realy unique: nature wings (dragon like with branches as bones decorated with flowers) and leave inspired membrane (same as my treant dragon I build over a year ago, it may be hard with just a few pixel but I share an example
![2020-05-14_18 17 17](https://user-images.githubusercontent.com/83673600/124509955-2267d480-ddd3-11eb-9e0e-91d2f2674300.png)
)
